models/br_ibge_censo_2022/br_ibge_censo_2022__populacao_residente_municipio.sql

```diff
@@ -12,12 +12,72 @@ with
             safe_cast(forma_de_declaracao_da_idade as string) forma_declaracao_idade,
             safe_cast(sexo as string) sexo,
             safe_cast(idade as string) idade,
+            case
+                when idade = 'Menos de 1 mês'
+                then 0
+                when idade = 'Menos de 1 ano'
```

Comment:
esse aqui não é também uma agregação de todas as colunas que tem o valos em meses ?

Reply:
verdade, preciso tirar. Ainda to conferindo e comparando os resultados com o ibge

Reply:
beleza, aguardando aqui